index.js

```diff
@@ -0,0 +1,223 @@
+import keys from './js/keys_data.js';
+
+class KeyboardClass {
+    constructor() {
+        this.enLang = localStorage.getItem('lang') || 'en';
+        this.shift = false;
+        this.caps = false;
+        this.keys = keys;
+    }
+
+    generateBaseStructure() {
+        const body = document.querySelector('body');
+        this.display = document.createElement('div');
+        this.display.classList.add('display');
+        this.wrapper = document.createElement('div');
+        this.wrapper.classList.add('wrapper');
+        this.textarea = document.createElement('textarea');
```

Comment:
Code duplication. Move repeated code logic in a separate method as we discussed.